packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -281,17 +291,16 @@ export class GCFBootstrapper {
       // Job was scheduled for a single repository:
       await this.scheduledToTask(body.repo, id, body, eventName, signature);
     } else {
-      const octokit = new Octokit({
-        auth: await this.getInstallationToken(body.installation.id),
-      });
+      const octokit = await this.getAuthenticatedOctokit(body.installation.id);
       // Installations API documented here: https://developer.github.com/v3/apps/installations/
-      const installationsPaginated = octokit.paginate.iterator({
-        url: '/installation/repositories',
-        method: 'GET',
-        headers: {
-          Accept: 'application/vnd.github.machine-man-preview+json',
-        },
-      });
+      const installationsPaginated = octokit.paginate.iterator(
+        octokit.apps.listReposAccessibleToInstallation,
+        {
+          headers: {
+            Accept: 'application/vnd.github.machine-man-preview+json',
+          },
```

Comment:
the `machine-man` preview was graduated ([announcement](https://developer.github.com/changes/2020-08-20-graduate-machine-man-and-sailor-v-previews/)). If you work with GHES, you can enable previews more easily like this
```suggestion
          mediaType: {
            previews: ['machine-man'],
          },
```